demo/vl-upload.html

```diff
@@ -107,6 +107,13 @@
             <vl-upload id="vl-upload-custom-text" url="http://httpbin.org/post" data-vl-input-name="files" id="vl-upload" data-vl-title="Afbeelding toevoegen" data-vl-sub-title="Sleep de afbeelding naar hier om toe te voegen"></vl-upload>
         </vl-demo>
 
+        <vl-demo data-vl-title="Upload met titel en sub-titel slot">
+            <vl-upload id="vl-title-sub-title-slot" url="http://httpbin.org/post" data-vl-input-name="files">
+                <span slot="title">Titel</span>
+                <span slot="sub-title">Sub-titel</span>
```

Comment:
subtitle/subtitel zou ik omschrijven als 1 woord (dus zonder - ) . Indien het voor breaking changes zorgt, zou ik het eerder wel zo laten.

Reply:
Sander had dit ook al aangehaald. Nu deze component heeft al een attribuut genaamd sub-title en ik had dit naamgebruik ook zonder verdere bedenkingen overgenomen. 

![afbeelding](https://user-images.githubusercontent.com/1375279/115229867-daaaa880-a113-11eb-9712-3ce9a4552898.png)

Het klopt wel dat het subtitle zou moeten zijn en ik meen dat dat in andere componenten dan weer wel zo gebruikt wordt. Nu omdat dit een breaking change is is er gezegd dit (voorlopig) zo te laten. Nu ik kan er wel een issue voor aanmaken zodat dit op de planning komt.